### ISSUE TITLE
Minepacks generates an error on startup but still works

### ISSUE THREAD

**Issue by AMDBartek** [GeorgH93/Minepacks#107]
### Information
**Environment information**
*Plugin + server version info*:

```
[20:53:15 INFO]: ##### Start Minepacks version info #####
[20:53:15 INFO]: Minepacks: 2.3.15-Release
[20:53:15 INFO]: PCGF PluginLib: 1.0.30-SNAPSHOT-T20200729160903
[20:53:15 INFO]: Server: git-Paper-1618 (MC: 1.12.2)
[20:53:15 INFO]: Java: 1.8.0_251
[20:53:15 INFO]: #####  End Minepacks version info  #####
```

*Online mode*: no 

*BungeeCord*: no 

**Server/crash log**

**Not a crash log, it is the error generated by Minepacks**
```
[20:51:12] [Server thread/ERROR]: [org.bukkit.craftbukkit.v1_12_R1.CraftServer] org.bukkit.plugin.PluginDescriptionFile.getProvides()Ljava/util/List; initializing Minepacks v2.3.15-Release (Is it up to date?)
java.lang.NoSuchMethodError: org.bukkit.plugin.PluginDescriptionFile.getProvides()Ljava/util/List;
	at at.pcgamingfreaks.Minepacks.Bukkit.BadRabbit.onLoad(BadRabbit.java:71) ~[?:?]
	at org.bukkit.craftbukkit.v1_12_R1.CraftServer.loadPlugins(CraftServer.java:323) ~[patched_1.12.2.jar:git-Paper-1618]
	at net.minecraft.server.v1_12_R1.DedicatedServer.init(DedicatedServer.java:222) ~[patched_1.12.2.jar:git-Paper-1618]
	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:616) ~[patched_1.12.2.jar:git-Paper-1618]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_251]
```
Here is a gist to the error link if preferred: https://gist.github.com/AMDBartek/8189cf1807522d5846051f1a2a325a84

**Plugin config (optional)**

```
Not a config issue
```

### Details
**Description**  

The plugin generates an error message in the log, but it still works fine.

**Steps to reproduce**  

Start the server with Minepacks installed

**Expected behavior**  

The plugin to start up with no errors.

**Other information** (e.g. detailed explanation, related issues, suggestions how to fix, links for us to have context, screenshots, etc.)
N/A

**Comment by GeorgH93** [comment]
Should be fixed with [v2.3.16](https://github.com/GeorgH93/Minepacks/releases/tag/v2.3.16).

**Comment by AMDBartek** [comment]
The fix has worked, thank you for fixing this issue.
